v2/emailpassword/serverless/with-aws-lambda/setup-lambda.mdx

```diff
@@ -20,27 +20,27 @@ import TabItem from '@theme/TabItem';
 - Click "Create Function" in the AWS Lambda dashboard, enter the function name and runtime, and create your Lambda function.
 <BackendSDKTabs disableGolang>
   <TabItem value="nodejs">
-    <img src="/img/integration-lambda/create-function-node.png"/>
+    <img src="/img/integration-lambda/create-function-node.png" alt="Create new Lambda configurations UI Node"/>
   </TabItem>
   <TabItem value="python">
-    <img src="/img/integration-lambda/create-function-python.png"/>
+    <img src="/img/integration-lambda/create-function-python.png" alt="Create new Lamda configurations UI Python"/>
```

Comment:
```suggestion
    <img src="/img/integration-lambda/create-function-python.png" alt="Create new Lambda configurations UI Python"/>
```

Reply:
Okay I see similar typos in all of the changed file as well can you fix that?

Reply:
done